### PR TITLE
Feature/mns detail

### PR DESCRIPTION
**Pull request by zhangjing28**
## Description

UI for Name Service Detail page

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [X] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [X] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [ ] Update the version number in `package.json` and `src/config/common.json`
- [ ] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [ ] Update `CHANGELOG.md`
- [ ] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

**Review by Kevingislason — COMMENTED** [review]
Code looks good, I just had one very small suggestion.

UI looks good, just needs a few tweaks to match the design perfectly:
- The "Available" / "Registered" / "Not Available" buttons are different colors in the design (on both the search page and the details page.
- The "Transfer" button is a slightly different color in the design, and has a different icon. You can get both the exact color and the icon itself from the figma mockup.
- The transfer modal has an icon next to "Click to modify" in the design
- The copy-paste icon has a slightly different color in the design

**Comment by zhangjing28** [comment]
> * The "Available" / "Registered" / "Not Available" buttons are different colors in the design (on both the search page and the details page

I checked the both codes and the design, and didn't see any difference here. The 'Available' button's color is `#2ee9a5`, the 'Registered' button's color is `#2ec7e9`, and the 'Not Available' button's color is `#f9413e`.

File: src/pages/NamePage/NameDetail.tsx [hunk 1]
```diff
@@ -1,9 +1,95 @@
...
+  } = useModal();
+  const handleCopy = () => {
+    copyRef?.current?.click();
+  };
   return (
-    <div>
-      <h1 className="text-3xl mb-0 pl-2 font-semibold text-accent">
-        Name Service Detail
-      </h1>
+    <div className="2xl:inset-x-0 flex flex-col flex-1 mx-10 mb-32 bg-secondary rounded-xl pb-6 ">
```
[the remark this answers is not in the document]
Comment (zhangjing28) [reply src/pages/NamePage/NameDetail.tsx:22]: Yeah, I agree with you. But I plan to do this when implementing the interaction with contracts. I think it's a better time.

File: src/pages/NamePage/NameDetail.tsx [hunk 2]
```diff
@@ -1,9 +1,106 @@
...
+              <span className="text-opacity-100 text-white">zk</span>
+            </p>
+            <p className="text-white text-opacity-60 text-sm flex justify-between m-4">
+              Registrant
+              <span className="text-opacity-100 text-white">000</span>
+            </p>
+            <p className="text-fourth flex justify-end m-4">
+              <button className="flex items-center" onClick={showModal}>
+                Transfer
+                <svg
```
Comment (zhangjing28) [inline src/pages/NamePage/NameDetail.tsx:81]: Will remove this `svg` tag to a svg file after refactor the icons

File: src/pages/NamePage/components/TransferModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            { disabled: disabled }
+          )}
+          ref={inputRef}
+          disabled={disabled}
+        />
+      </p>
+      <div
+        className="flex items-center ml-24 cursor-pointer"
+        onClick={handleFocus}>
+        <svg
```
Comment (zhangjing28) [inline src/pages/NamePage/components/TransferModal/index.tsx:40]: This `svg` tag will remove after icon refactor too.

**Comment by Kevingislason** [comment]
> I checked the both codes and the design, and didn't see any difference here. The 'Available' button's color is `#2ee9a5`, the 'Registered' button's color is `#2ec7e9`, and the 'Not Available' button's color is `#f9413e`.

This is how the buttons appear to me, do they appear different for you?

Screen Shot 2023-01-12 at 8 20 40 PM
Screen Shot 2023-01-12 at 8 14 25 PM

**Comment by zhangjing28** [comment]
> > I checked the both codes and the design, and didn't see any difference here. The 'Available' button's color is `#2ee9a5`, the 'Registered' button's color is `#2ec7e9`, and the 'Not Available' button's color is `#f9413e`.
> 
> This is how the buttons appear to me, do they appear different for you?
> 
> Screen Shot 2023-01-12 at 8 20 40 PM
> 
> Screen Shot 2023-01-12 at 8 14 25 PM

It's works fine in my environment
image

**Comment by Kevingislason** [comment]
The colors looks as expected for me locally, but not on the vercel deployment

**Comment by zhangjing28** [comment]
> The colors looks as expected for me locally, but not on the vercel deployment

I think I find the problem. I set the `className` here like this 
```
className={`border-manta-${clsMap[status]} text-manta-${clsMap[status]}`}
```
It caused something wrong with the CSS files in the production environment， there are no related styles to `.border-manta-success` or `.text-manta-success`😭
Let me try to fix it. Also if it happens to you before, it's would be nice to share with me

**Pull request closed without merging**